sdk/src/main/java/io/opentelemetry/sdk/trace/SpanBuilderSdk.java

```diff
@@ -221,6 +221,18 @@ private static SpanContext parent(
     throw new IllegalStateException("Unknown parent type");
   }
 
+  @Nullable
+  private static Span parentSpan(ParentType parentType, Span explicitParent) {
```

Comment:
It feels a bit weird to have `parent` and `parentSpan` could be there a way to unify them together?

Reply:
Yes, hence my comment about having them 'together' ;)